db/src/Pos/DB/Block/Epoch.hs

```diff
@@ -380,6 +433,11 @@ packWord32 = LBS.toStrict . encode
 unpackWord32 :: ByteString -> Word32
 unpackWord32 = decode . LBS.fromStrict
 
+sleepSeconds :: MonadIO m => Int -> m ()
+sleepSeconds sec =
+    -- 'threadDelay' uses microseconds
+    liftIO . threadDelay $ sec * 1000 * 1000
```

Comment:
note that this will not work for a 24 hour interval on 32 bit machines - 2^31 microseconds is just shy of 36 minutes.

Reply:
[This?](https://hackage.haskell.org/package/concurrent-extra-0.5/docs/Control-Concurrent-Thread-Delay.html)

Reply:
We don't currently build for 32 bit machines do we?

Reply:
Yep, good idea!

Reply:
Wow, `delay` has been removed from the current version of that package. Fortunately its onl about 4 lines of code.